CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -1,8 +1,24 @@
+# [v1.10.2](https://github.com/kubermatic/kubeone/releases/tag/v1.10.2) - 2025-07-21
+
+## Changelog since v1.10.1
+
+## Changes by Kind
+
+### Chore
```

Comment:
All updates should go under the `Updates` section, see prior changelogs.

Reply:
`release-note` figured this kind from the PRs label, I suppose we should swap kind/chor to kind/updates in that case.

Reply:
I agree, I wanted to do that, but I couldn't find time 😅 It should be relatively easy though